packages/data/lib/src/repositories/date_preferences_repository.dart

```diff
@@ -0,0 +1,45 @@
+import 'package:domain/domain.dart';
+import 'package:shared_preferences/shared_preferences.dart';
+
+const _trendingUpdateDateKey = 'trendingUpdateDate';
+const _anticipatedUpdateDateKey = 'anticipatedUpdateDate';
+
+class DatePreferencesRepositoryImpl implements DatePreferencesRepository {
```

Comment:
create separate table for dates. it's ok to have preferences, but let's keep more consistent flow (try to keep as much we can in the db)